README.md

```diff
@@ -62,7 +66,7 @@ func initRouteRegistry(
 ```
 
 ### Configuration
-```
+```go
```

Comment:
This  `Configuration` section is maybe a bit confusing to users. 
Maybe add more detail on its usage and which instance will consume it?

Reply:
I think the major purposes is to show what is the configuration about. Usually usage of configuration would not be provided since they are just 'configurations'